File: API/SchoolNotyfyAPI_NEW/SchoolNotify.Domain/Entities/Aluno.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SchoolNotify.Domain.Entities
{
    public class Aluno
    {
        public int Id { get; set; }
        public int IdResponsavel { get; set; }
        public int IdSala { get; set; }
        public string Nome { get; set; }
        public int Matricula { get; set; }
        public int Idade { get; set; }
        public string Sexo { get; set; }

        public virtual Responsavel Responsavel { get; set; }
        public virtual Sala Sala { get; set; }
    }
}

File: Angular/SN-Ionic/src/models/aluno.ts
import { Responsavel } from "./responsavel";
import { Sala } from "./sala";

export class Aluno {
    id: number = 0;
    idResponsavel: number = 0;
    idSala: number = 0;
    nome: string = "";
    sexo: string = "";
    idade: number = null;

    responsavel: Responsavel = new Responsavel();
    sala: Sala = new Sala();

    constructor(values: Object = {}) {
        Object.assign(this, values);
    }
}